Update app-service-linux-faq.md

lastcoolnameleft opened the pull request:
After spending time debugging why my dependencies weren't loading, I discovered that the app uses composer.lock NOT composer.json to include new packages.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-diree). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Blackmist commented:
looks good, thanks.

lastcoolnameleft committed:
Update app-service-linux-faq.md

After spending time debugging why my dependencies weren't loading, I discovered that the app uses composer.lock NOT composer.json to include new packages.
